_posts/2024-08-07-taming-apache-kafka-4.0.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: post
+title:  "Taming Apache Kafka 4.0"
```

Comment:
Title is engaging, but I wonder if "taming" implies Kafka 4.0 is difficult or problematic?  A couple of suggestions: "What Apache Kafka 4.0 means for Strimzi", "How Apache Kafka 4.0 impacts Strimzi", "Preparing for Apache Kafka 4.0", "Getting ready for Apache Kafka 4.0"

Reply:
> Title is engaging, but I wonder if "taming" implies Kafka 4.0 is difficult or problematic?

That was pretty much the intention. It is definitely difficult given the amount and impact of the changes, the missing features, the _delays_ - both for us and for all users. I expect that it will turn out to be problematic as well, but that is so far just my expectation and remains to be seen 😮.

Reply:
I guess it's good to be upfront about the "challenges" ahead. :smiling_face_with_tear: